packages/components/src/visually-hidden/style.scss

```diff
@@ -0,0 +1,30 @@
+.components-visually-hidden {
+	border: 0;
+	clip: rect(1px, 1px, 1px, 1px);
+	-webkit-clip-path: inset(50%);
+	clip-path: inset(50%);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+	word-wrap: normal !important;
+}
+
+.components-visually-hidden:focus {
+	background-color: #eee;
```

Comment:
Can this be converted to one of the variables used in SCSS files?

Reply:
This raises another issue, it doesn't look like the SCSS variables are defined within Components package, or if they are I've missed where. It seems they rely on Block Editor styling, which can be used, but not sure where this separation should be. So this might need to become a separate issue for how to define and use SCSS with Components.

Reply:
That's an already solved problem. variables are now defined in the `base-styles` package and these are made available to all packages using SASS including the @wordpress/components package.

Reply:
:+1: Great. So sounds like mostly a matter of documentation and examples for the case that someone wants to use the components package, show they would define their own or use base-styles package.